sdk/communication/Azure.Communication.Chat/src/Models/PhoneNumberIdentifierModel.cs

```diff
@@ -6,7 +6,7 @@
 namespace Azure.Communication
 {
     [CodeGenModel("PhoneNumberIdentifierModel")]
-    internal partial class PhoneNumberIdentifierModel
+    public partial class PhoneNumberIdentifierModel
```

Comment:
`internal`

Reply:
set to public since CommunicationIdentifierModel needs to be public (reasoning as above)